tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -2062,13 +2062,12 @@ public function userHasCopiedOrMovedFileInsideSpaceUsingFileId(
 	): void {
 		// split the source when there are sub-folders
 		$sourceFile = \trim($sourceFile, "/");
-		$sourceFile = \explode("/", $sourceFile);
-		$sourceFile = \end($sourceFile);
 		$destinationFile = \trim($destinationFile, "/");
-		if ($actionType === 'copied' || $actionType === 'moved') {
-			$fileDestination = $this->escapePath($destinationFile) . '/' . $this->escapePath($sourceFile);
-		} else {
-			$fileDestination = $destinationFile;
+
+		$fileDestination = $this->escapePath($destinationFile);
+
+		if ($actionType !== 'renamed' && !strpos($destinationFile, "/")) {
+			$fileDestination .= '/' . $this->escapePath($sourceFile);
```

Comment:
we should be able to revert these changes now

Reply:
I think we will be needing:
```php
if (!strpos($destinationFile, "/")) {
			$fileDestination .= '/' . $this->escapePath($sourceFile);
		}
```
for steps like:
```gherkin
    And user "Alice" has moved file "textfile.txt" into "FOLDER/renamed.txt" inside space "Personal" using file-id "<<FILEID>>"
```
because we have to append destination file name here